docs/user_guide/spin_system/spin_system.rst

```diff
@@ -4,10 +4,56 @@
 Spin System
 ===========
 
-At the heart of any mrsimulator calculation is the definition of a :ref:`spin_sys_api`
-object describing the sites and couplings within a spin system. Each :ref:`simulator_api` object
-holds a list of :ref:`spin_sys_api` objects which will be used to calculate frequency
-contributions.
+At the heart of any ``mrsimulator`` calculation is the definition of
+a :ref:`spin_sys_api` object describing the sites and couplings within a spin
+system. Each :ref:`simulator_api` object holds a list of :ref:`spin_sys_api`
+objects which are used to calculate frequency contributions.
+
+.. note::
```

Comment:
@pjgrandinetti Although the information in this note block is important, the note itself is quite long. Notes should be for a sentence or brief paragraph. Perhaps a new sub-section titled **Theoretical Background** with this content

Reply:
wait til you see what I did in Method.rst.   

Probably could take this out of note.

Reply:
Yes I saw the transition symmetry function note yesterday. will make a separate comment for that